documents/motions/10.2022.4 - Ethics Committee.md

```diff
@@ -1,35 +1,25 @@
-**Date:** May 1, 2022  
+**Date:** TBC, 2024  
 
 # Motion
 
-The WCA Ethics Committee is an Advisory Committee of the WCA.
+The WCA Appeals Committee is an Advisory Committee of the WCA.
 
-1. The WCA Ethics Committee has the following rights and duties:
-   1. Maintaining and publishing a Code of Ethics that all WCA Staff must abide by.
-   2. Functioning as the Board of Appeal if WCA Staff or Registered Speedcubers do not agree with the conclusions and advices of the WCA Disciplinary Committee regarding investigations related to the persons involved.
-      1. The appeal must be initiated by the WCA Board on behalf of the affected individuals. WCA Board must not deny the initiation of appeals made by affected individuals.
-      2. The WCA Ethics Committee may refuse appeals if the Committee decides there is no valid ground for appeal.
-   3. Performing independent investigations regarding WCA Staff
-      1. The subject of the investigations is any conduct of WCA Staff that is not compliant with the Ethics codes or the Spirit of the WCA.
-      2. The investigations can be initiated by the WCA Ethics Committee itself, by the WCA Disciplinary Committee or by the WCA Board.
-   4. Making decisions on disciplinary actions in relation to the investigated incidents.
-   5. Informing the WCA Board on investigations, decisions, and disciplinary actions.
-   6. Publishing public information on the investigations, decisions, and disciplinary actions where necessary.
-   7. Managing an administration of the investigations, the reports, the decisions, and the disciplinary actions.
-   8. Being a confidant to WCA Staff and Registered Speedcubers regarding ethical matters and questions within the WCA.
-   9. WCA Staff and Registered Speedcubers must never start communications with the WCA Ethics Committee regarding matters related to the WCA Disciplinary Committee or the WCA Ethics Committee, except for the following matters:
-      1. Requests for investigations by the WCA Board and the WCA Disciplinary Committee.
-      2. Requests for appeal by the WCA Board on behalf of a Registered Speedcuber or WCA Staff.
-      3. Financial Support of the WCA Ethics Committee by the WCA Financial Committee.
-      4. Questions and reports to the confidant within the WCA Ethics Committee.
-2. All WCA Staff and Registered Speedcubers must fully cooperate with investigations by the WCA Ethics Committee.
-3. In case the WCA Ethics Committee is not constituted or for any reason non-functional, the WCA Disciplinary Committee will act as WCA Ethics Committee.
-4. The WCA Ethics Committee will take the role of WCA Disciplinary Committee in case the WCA Disciplinary Committee is not constituted or for any reason non-functional.
+1. The WCA Appeals Committee has the following rights and duties:
+   1. Functioning as the Board of Appeal in line with the processes described in [WCA Motion : Disputes](wca{documents/motions/16.2021.1 - Disputes.pdf}).
```

Comment:
Do we want the Appeals Committee to function as the Board of appeal, or have its members form the Board of appeal? If you want the WApC to be attractive to experienced WCA Staff who don't want to do heaps of work maybe having a system where they don't have to handle every case would be better, and you could have say 3 members as the Board of Appeal for a given case. This would also allow them to specialize in certain types of appeals.

Reply:
Added